deploy/future

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Don't run this with -e, grep exits with 1 when it finds no results
+
+function check_py () {
+# This is a bit janky. If you know how to get the output
+# to go on separate lines, please tell me.
+    results=`git --no-pager grep $1|grep "\.py"|grep -v "future-ok"`
+    if [ "$results" ]; then
+	echo $results
+	EXIT=1
+    fi
+}
+
+function check_html () {
+    results=`git --no-pager grep $1|grep "\.html:"|grep -v "future-ok"`
+    if [ "$results" ]; then
+	echo $results
+	EXIT=1
+    fi
+}
+
+EXIT=0
+
+# http://stackoverflow.com/a/4774063
+pushd "$(dirname "$0")" > /dev/null
+SCRIPTDIR="$(pwd -P)"
+popd > /dev/null
+
+BASEDIR="$(dirname "$SCRIPTDIR")"
+cd $BASEDIR
+
+echo "Running future Django lint checks..."
+
+# Uncomment checks here once you've deleted all the uses from the codebase,
+# or checked that we never used the feature in the first place,
+# or checked that future changes won't break your use of this feature.
+# Add more checks once a new version of Django is released,
+# and delete them once we've upgraded main to a new version and
+# cleared out old pull requests.
+# Only push to future-django-support if what you're doing won't work
+# on our current Django version. Otherwise, make a pull request to
+# main and add the check as part of your pull request.
+
+# A lot of these are impossible or annoying to write without some
+# false positives, and some ungreppable things are left out entirely.
+# Many of them should be considered warnings rather than errors: check
+# that a future behavior change won't break your code.
+# Add "#future-ok" to the end of a line if you've checked,
+# or rewrite the check to avoid the false positives.
+
+# Django 1.9 backwards incompatibility checks
+#check_py is_usable
+#check_py "\.add(" # warning
```

Comment:
Yeah, and it's not the only one like that in here. In this case, the list of results isn't too long to look at by hand, so I figure it's more useful to leave this in here as a reminder. Some of the others will need to be rewritten if they're going to be at all useful, though.